client/components/form-elements/FormControl.tsx

```diff
@@ -0,0 +1,16 @@
+import { FormControl as ChakraFormControl } from '@chakra-ui/react';
+import type { FormControlProps } from '@chakra-ui/react';
+import { FieldErrors } from 'react-hook-form';
+type Props = {
```

Comment:
Very trivial but for readable purposes which it would somehow separate type declarations and imports, add a spacing here

Thus
```ts
import ...

type Props = {
```